pom.xml

```diff
@@ -551,6 +551,12 @@
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
+
+            <dependency>
+                <groupId>com.maxmind.geoip2</groupId>
+                <artifactId>geoip2</artifactId>
+                <version>2.16.1</version>
+            </dependency>
```

Comment:
Rather than promoting this version to the root Maven configuration, it seems better to add it in the `pom.xml` for `nifi-nar-bundles`.  That should cover all current dependencies, and that Maven configuration already has a dependency management section. If this dependency extends to other components, such as MiNiFi or Registry, then we could consider moving it to the root Maven configuration at that time.

Reply:
Done.